src/index.ts

```diff
@@ -1,22 +1,24 @@
 import { JwtPayload } from 'jsonwebtoken'
 import { createJWKS, createKeyPair, signJwt } from './tools.js'
 import { setupServer } from 'msw/node'
-import { rest } from 'msw'
+import { HttpResponse, http, RequestHandlerOptions, HttpHandler } from 'msw'
 
 const createJWKSMock = (
   jwksBase: string,
   jwksPath = '/.well-known/jwks.json'
-) => {
+): JWKSMock => {
   const keypair = createKeyPair()
   const JWKS = createJWKS({
     ...keypair,
     jwksOrigin: jwksBase,
   })
-  const server = setupServer(
-    rest.get(new URL(jwksPath, jwksBase).href, (_, res, ctx) =>
-      res(ctx.status(200), ctx.json(JWKS))
+  const jwksHandler = (options?: RequestHandlerOptions) =>
```

Comment:
Why allow for providing [options](https://mswjs.io/docs/api/http#handler-options) here? The only thing one could do it with it is to make a "one time handler" which I find kinda pointless. If the support for injecting options would be dropped, this would not need to be a function any more but could be a property.

Reply:
Yeah I started with it as a property but was trying to make it flexible - e.g. might want to ensure only one request is ever made. I don't currently use that option though so might be overkill. I also saw `kid` was a function so I kept a similar interface.

Reply:
Ha, you are right. Another unnecessary function. Possibly I was concerned about people writing to these variables. But I guess that is also a bit paranoid. I will provide the values for now but keep the `kid` as it is to not break the interface.